src/components/Accordion.js

```diff
@@ -24,6 +24,38 @@ document.addEventListener(
   false
 );
 
+document.addEventListener(
+  "focusin",
+  onAccordionFocus => {
+    const { target } = onAccordionFocus;
+    changeFocusClass(target, "open");
+  },
+  false
+);
+
+document.addEventListener(
+  "focusout",
+  onAccordionFocus => {
+    const { target } = onAccordionFocus;
+    changeFocusClass(target, "close");
+  },
+  false
+);
+
+const changeFocusClass = (target, status) => {
+  var ua = window.navigator.userAgent;
+  var isIE = /MSIE|Trident|Edge\//.test(ua);
+  if (isIE) {
+    var accordionHeaderText = target.getElementsByClassName(
+      "dg_accordion_buttontext-holder"
+    );
+    status === "open"
```

Comment:
As we talked about in person, this doesn't really need to be a ternary. But the important thing here, is that you can access the classList here to simplify this code.

I think this should work

```js
accordionHeaderText[0].classList[status === "open" ? "add" : "remove"]("ms-focus-withing"). 
```

This will add or remove the status based on status being equal to open.

This is equivalent to essentially

```js
if (status === "open") {
  accordionHeaderText[0].classList.add("ms-focus-within");
}
else {
  accordionHeaderText[0].classList.remove("ms-focus-within");
}
```

Reply:
@sgrg1 @martypowell this is happening in Chrome as well. It appears to not like the open and close all actions sometimes. It looks like its some kind of race condition which makes me a sad panda.

Reply:
@tmccoy529 , ignored, not sure what it was. Marking as resolved.